components/layout/stylesheets/servo.css

```diff
@@ -25,37 +25,38 @@ input::-servo-text-control-inner-editor {
 }
 
 input::-servo-text-control-inner-container {
-  position: relative;
+  display: flex;
   height: 100%;
   pointer-events: none;
-  display: flex;
+  position: relative;
 }
 
 input:not(:placeholder-shown)::placeholder {
-  visibility: hidden !important
+  visibility: hidden !important;
 }
 
 input::-servo-text-control-inner-editor, input::placeholder {
-  white-space: pre;
-  margin-block: auto !important;
-  inset-block: 0 !important;
   block-size: fit-content !important;
+  inset-block: 0 !important;
+  margin-block: auto !important;
+  min-width: 100%;
```

Comment:
Maybe `min-width: stretch` is better in case we add some padding or such?
But is there a reason to use flex layout? If it was a block-level it would stretch by default.
Or since it's a flex item, maybe `flex: 1` could make more sense. Maybe with `min-width: 0` to avoid overflow?